generate_stats.py

```diff
@@ -5,40 +5,65 @@
 sys.path.append('/opt/tljh/user/lib/moab/lib/python3.6/site-packages/pymoab-5.1.0-py3.6-linux-x86_64.egg')
 from pymoab import core, types
 from pymoab.rng import Range
-
+import matplotlib.pyplot as plt
+import numpy as np
 # import the new module that defines each of the functions
 import dagmc_stats
+import argparse
+
+
+
+
 
-def report_stats(entity_ranges):
+def report_stats(entity_ranges, surface_per_volume_stats, triangle_per_vertex_stats, verbose):
```

Comment:
As the list of different stats grows, we should discuss the design of the data structures.  We probably want to switch to an overarching class or dictionary.